src/toil/batchSystems/slurm.py

```diff
@@ -193,6 +193,15 @@ def prepareSbatch(self, cpu, mem, jobID):
         if cpu is not None:
             sbatch_line.append('--cpus-per-task={}'.format(int(math.ceil(cpu))))
 
+        # "Native extensions" for SLURM (see DRMAA or SAGA)
+        nativeConfig = os.getenv('TOIL_SLURM_ARGS')
+        if nativeConfig is not None:
+            logger.debug("Native SLURM options appended to sbatch from TOIL_SLURM_RESOURCES env. variable: {}".format(nativeConfig))
```

Comment:
Was just trying to future-proof it:

https://pyformat.info/

`%s` is old format.